recipe/build.sh

```diff
@@ -8,26 +8,81 @@ else
   export LDFLAGS="${LDFLAGS} -lrt"
 fi
 
-if [[ "${target_platform}" == linux-aarch64 ]]; then
-  echo "TODO debug why using gen-bazel-toolchain leads to undeclared inclusion(s) of pybind11"
+export LD_LIBRARY_PATH="${LD_LIBRARY_PATH+LD_LIBRARY_PATH:}:$PREFIX/lib"
+
+# Build with clang on OSX-*. Stick with gcc on linux-*.
+if [[ "${target_platform}" == linux-* ]]; then
+  export BUILD_FLAGS="--use_clang=false"
 else
+  export BUILD_FLAGS="--use_clang=true --clang_path=${BUILD_PREFIX}/bin/clang"
+fi
+
 source gen-bazel-toolchain
 
 cat >> .bazelrc <<EOF
+
+build --copt=-isysroot${CONDA_BUILD_SYSROOT}
+build --host_copt=-isysroot${CONDA_BUILD_SYSROOT}
+build --linkopt=-isysroot${CONDA_BUILD_SYSROOT}
+build --host_linkopt=-isysroot${CONDA_BUILD_SYSROOT}
 build --crosstool_top=//bazel_toolchain:toolchain
 build --logging=6
 build --verbose_failures
 build --toolchain_resolution_debug
 build --define=PREFIX=${PREFIX}
 build --define=PROTOBUF_INCLUDE_PATH=${PREFIX}/include
 build --local_cpu_resources=${CPU_COUNT}"
+build --define=with_cuda=false
+build --cxxopt=-I${PREFIX}/include
```

Comment:
🤔 Isn't it already set by `bazel-toolchain`?

Reply:
This might have been added before I got the toolchain working on aarch64. I'll remove it and confirm.